### PR TITLE
Yautja Actions Update (/QOL changes)

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

This was hell

Action sprites from https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps/pull/206

# Explain why it's good for the game

Ease of use

# Changelog

:cl: BeagleGaming1, Mr.Crowford
add: Added Yautja action buttons
qol: Added a keybind to control falcon drones
qol: Thwei crystals and healing capsules say how much time until off cooldown
code: Yautja keybinds now redirect to action buttons
code: Changed thwei crystals and healing capsules to cooldowns instead of timers
code: Miscellaneous Yautja-related code cleanup
/:cl:

### CODE REVIEW

File: code/modules/cm_preds/yaut_actions.dm [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+			path = text2path(params["emotePath"])
+
+			if(!path || !COOLDOWN_FINISHED(src, panel_emote_cooldown))
+				return
+
+			if(!(path in subtypesof(/datum/emote/living/carbon/human/yautja)))
+				return FALSE
+
+			COOLDOWN_START(src, panel_emote_cooldown, 2.5 SECONDS)
+			usr.emote(initial(path.key))
```
[the remark this answers is not in the document]
Comment (Cthulhu80) [reply code/modules/cm_preds/yaut_actions.dm:326]: https://github.com/cmss13-devs/cmss13/pull/3101

File: code/modules/cm_preds/yaut_actions.dm [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+	var/image/new_overlays
+	new_overlays = image(icon_file, button, new_icon_state)
+
+	button.overlays += new_overlays
+
+//Actions that require wearing bracers
+/datum/action/predator_action/bracer
+	require_bracers = TRUE
+
+/datum/action/predator_action/bracer/control_falcon_drone
```
Comment (Drulikar) [inline code/modules/cm_preds/yaut_actions.dm:143]: Shouldn't this also require visors?

File: code/modules/cm_preds/yaut_weapons.dm [hunk 3]
```diff
@@ -278,6 +280,10 @@
 	return TRUE
 
 /obj/item/weapon/yautja/combistick/proc/setup_chain(mob/living/user)
+	give_action(user, /datum/action/predator_action/bracer/combistick)
+	add_verb(user, /mob/living/carbon/human/proc/call_combi)
+	linked_to = user
+
 	var/list/tether_effects = apply_tether(user, src, range = 6, resistable = FALSE)
```
Comment (Drulikar) [inline code/modules/cm_preds/yaut_weapons.dm:287]: This from dropping a combistick results in `/datum/effects/tethering/proc/set_tethered(datum/effects/tethered/T)` throwing `Exception has occurred: Cannot execute null.beam().` when self destructing
image

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Review by nauticall — APPROVED** [review]
sprites good

**Review by Drulikar — APPROVED** [review]
All issues appear resolved and nothing else sticks out to me as needing changing.

The only thing else you may consider is using hide for the actions instead of add/remove so their positions are more consistent.

**Pull request merged**